DQM/HLTEvF/plugins/HLTObjectsMonitor.cc

```diff
@@ -481,7 +481,7 @@ void HLTObjectsMonitor::analyze(const edm::Event& iEvent, const edm::EventSetup&
               if (key != key1 &&
                   kCnt1 > kCnt0) {  // avoid filling hists with same objs && avoid double counting separate objs
 
-                double pt2 = objects[key1].phi();
+                double pt2 = objects[key1].pt();
                 double eta2 = objects[key1].eta();
                 double phi2 = objects[key1].phi();
                 int id2 = objects[key1].id();
```

Comment:
One note (maybe just to myself) which goes a bit beyond this PR.

I was trying to figure out why the [original warning](https://github.com/cms-sw/cmssw/pull/23185#issue-322013542) quoted `89` and `0` as "object IDs", while the trigger-IDs of the objects created by `HLTHtMhtFilter` are `89` and `90` [[1]](https://github.com/cms-sw/cmssw/blob/ff75956ff1b3cf2f1ccdda06e31f739fc447f798/HLTrigger/JetMET/src/HLTHtMhtFilter.cc#L104-L105) [[2]](https://github.com/cms-sw/cmssw/blob/ff75956ff1b3cf2f1ccdda06e31f739fc447f798/DataFormats/HLTReco/interface/TriggerTypeDefs.h#L85-L86).

The answer is in the implementation of `TriggerSummaryProducerAOD`, and the bottom line is that one can get the correct IDs using `TriggerEvent::filterIds` (in analogy to `TriggerEvent::filterKeys`), instead of relying on `TriggerObject::id` (which is what is currently done in this DQM module). Martin can correct me if I misunderstood this.